File: StructuralDesignPattern/BridgePattern/java/Driver.java
package StructuralDesignPattern.BridgePattern.java;

import StructuralDesignPattern.BridgePattern.java.Abstractions.Circle;
import StructuralDesignPattern.BridgePattern.java.Abstractions.Square;
import StructuralDesignPattern.BridgePattern.java.Implementors.BlueColor;
import StructuralDesignPattern.BridgePattern.java.Implementors.RedColor;

public class Driver {
    public static void main(String[] args){
        Square redsquare = new Square(new RedColor());
        redsquare.draw();

        Circle bluecircle = new Circle(new BlueColor());
        bluecircle.draw();
    }
}

File: StructuralDesignPattern/BridgePattern/python/Driver.py
from Implementors.RedColor import RedColor
from Abstractions.Square import Square
from Abstractions.Circle import Circle
from Implementors.BlueColor import BlueColor


if __name__ == "__main__":

    square = Square(RedColor())
    square.draw()

    circle = Circle(BlueColor())
    circle.draw()